java-client-model/src/main/java/com/cumulocity/rest/representation/reliable/notification/NotificationTokenRequestRepresentation.java

```diff
@@ -0,0 +1,49 @@
+package com.cumulocity.rest.representation.reliable.notification;
+
+import com.cumulocity.rest.representation.AbstractExtensibleRepresentation;
+import com.cumulocity.rest.representation.annotation.Command;
+import com.cumulocity.rest.representation.annotation.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.svenson.JSONProperty;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Setter
+public class NotificationTokenRequestRepresentation extends AbstractExtensibleRepresentation {
+
+    public NotificationTokenRequestRepresentation(String subscriber, String subscription, long expiresInMinutes, boolean shared){
+        this(subscriber, subscription, null, true, expiresInMinutes, shared, false);
+    }
+
+    public NotificationTokenRequestRepresentation(final String subscriber, final String subscription, final long expiresInMinutes, final boolean shared, final boolean nonPersistent) {
+        this(subscriber, subscription, null, true, expiresInMinutes, shared, nonPersistent);
+    }
+
+    @NotNull(operation = {Command.CREATE})
+    @Getter(onMethod_ = @JSONProperty(ignoreIfNull = true))
```

Comment:
I wonder about all those annotations, if we need them in this module, like `operation = {Command.CREATE}`. 
This is a generic question.

Reply:
All javax validation annotations can be removed from here together with this one, again would be better to do it in separate PR in my opinion.
TODO